src/app/notes/notes.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'notes',
+  templateUrl: './notes.component.html',
+  styleUrls: ['./notes.component.css']
+})
+export class NotesComponent implements OnInit {
```

Comment:
Should this component allow you to edit the note?

Reply:
Yes, the idea is that the notes component is always editable (and resubmitted back up to the DB

Reply:
Yeah initially it was readonly, but after discussing it with the others it was decided to make it editable